# start or keep
load_model: false
model_name: dqn_tb3

# Env
CARLA_TIMEOUT: 20.0

# DRL Agent
learning_rate: 0.001
batch_size: 64
current_episode: 0
num_episodes: 10
episode_step: 10000
update_target_every: 1000
save_model_every: 500
aggregate_stats_every: 10
epsilon: 1
epsilon_decay: 0.995
min_epsilon: 0.1
discount_factor: 0.99

# Logs
tensorboard_base_log_dir: logs
SHOW_QS_VALUES: true

# Miscelanious
log_level: DEBUG
